app/routes/report/expressionVariants.js

```diff
@@ -86,7 +86,16 @@ router.route('/')
         include: [
           {
             model: db.models.kbMatches,
-            attributes: ['ident', 'category'],
+            attributes: ['ident'],
+            include: [
+              {
+                model: db.models.kbMatchedStatements,
+                as: 'kbMatchedStatements',
+                attributes:
+                  ['category'],
```

Comment:
(noting that this and the same change in copyVariants.js will probably need some updates in the client to handle)